qcfractal/interface/collections/dataset_view.py

```diff
@@ -1,15 +1,19 @@
 import abc
 import distutils
-import json
+import pathlib
 import warnings
 from contextlib import contextmanager
+from typing import Any, Dict, List, Tuple, Union
 
-from .dataset import Dataset
-import pathlib
-from typing import Union, List, Tuple
+import h5py
```

Comment:
Would it be feasible to make h5py optional in a `qcportal-core` or the like? It doesn't seem to be that way, but perhaps worth checking in on.

Reply:
I will revisit this when I get to `.download()` TODO comment added.